plugins/rtmp-services/data/services.json

```diff
@@ -235,6 +235,11 @@
                 "YouTube - RTMP",
                 "YouTube - RTMPS (Beta)"
             ],
+            "supported video codecs": [
+                "h264",
+                "hevc",
+                "av1"
+            ],
```

Comment:
You need to do the opposite. You need to add the following to very RTMP/RTMPS service:
```
            "supported video codecs": [
                "h264"
            ],
```

To avoid adding HEVC and AV1 support yo every RTMP/RTMPS services.

Reply:
It’s not the ‘opposite’ per-se, but I agree that we need to make this explicit (should’ve been in the first place, ‘assuming’ stuff from a lack of info is not a great design choice). As such we need to make that explicit now and have services explicitly state if they only support h264.

Reply:
@PatTheMav, ` "supported video codecs"` is meant to enable only a subset of what the protocol/output can handle (e.g. only H264 on HLS). Until now, RTMP was H264 only for streaming services.

So HEVC and AV1 are suddenly added to RTMP support and all RTMP services added until now only support H264 since it was the "standard".

Reply:
I understood that, but that means the configuration so far was very much tailored to a "there is only RTMP and thus there is only H264" mindset. Instead the mindset should be "services support specific codecs" and vanilla RTMP just happens to support H264 only, which should not be implicitly assumed by a config file or code parsing it.

The more pertinent question is whether we can add those "explicit" entries on master already, so we don't burden this PR with that cleanup work?

Reply:
Thanks for spotting, let us know, happy to do add them explicitly or wait for it on master.

Reply:
Looks like this is done now, thanks.